src/SDKs/Media/Media.Tests/ScenarioTests/StreamingLocatorTests.cs

```diff
@@ -76,12 +80,22 @@ internal static void ValidateLocator(StreamingLocator locator, string expectedNa
         {
             Assert.Equal(expectedAssetName, locator.AssetName);
             Assert.Equal(expectedName, locator.Name);
-            Assert.Empty(locator.ContentKeys);
+            //Assert.NotEmpty(locator.ContentKeys);  // TODO: This is currently not implemented consistently.  Verify it once it is
```

Comment:
nit: Please delete this line instead of commenting

Reply:
Removed.